avro-converter/src/main/java/io/confluent/connect/avro/AvroConverter.java

```diff
@@ -106,7 +106,7 @@ public SchemaAndValue toConnectData(String topic, byte[] value) {
       }
       throw new DataException("Unsupported type returned by deserialization");
     } catch (SerializationException e) {
-      throw new DataException("Failed to deserialize data to Avro: ", e);
+      throw new DataException("Failed to deserialize data for topic %s to Avro: ".format(topic), e);
```

Comment:
Do you want to get the one 2 lines up as well? Not exactly a common issue, but should probably just be consistent.